samples/snippets/table_sample.py

```diff
@@ -35,7 +35,7 @@ def table_sample(document_path: str, output_file_prefix: str) -> None:
             # Write table to CSV file
             output_file = f"{output_file_prefix}-{page_number}-{table_number}.csv"
```

Comment:
Maybe rename these `*_number` to `*_index`.

Reply:
Ah, this should be changed to use the actual `Page.page_number` instead of a locally created index.